src/data.rs

```diff
@@ -234,6 +243,7 @@ ast_struct! {
 }
 
 #[cfg(feature = "parsing")]
+#[cfg_attr(syn_enable_doc_cfg, doc(cfg(feature = "parsing")))]
```

Comment:
This module is marked `pub`, but isn't actually exposed from the crate, it's only exposed internally.
We don't need to add `doc` attributes onto them.